tuf/api/metadata.py

```diff
@@ -891,18 +852,14 @@ class DelegatedRole(Role):
 
     Attributes:
         name: A string giving the name of the delegated role.
-        keyids: A set of strings each of which represents a given key.
-        threshold: An integer representing the required number of keys for that
-            particular role.
         terminating: A boolean indicating whether subsequent delegations
-            should be considered.
+            should be considered during a target lookup.
         paths: An optional list of strings, where each string describes
-            a path that the role is trusted to provide.
+            a path pattern that the role is trusted to provide.
         path_hash_prefixes: An optional list of HEX_DIGESTs used to succinctly
             describe a set of target paths. Only one of the attributes "paths"
-            and "path_hash_prefixes" is allowed to be set.
+            and "path_hash_prefixes" is allowed within a DelegatedRole.
```

Comment:
I wonder if there's a clearer way to document that paths and paths_hash_prefixes are mutually exclusive?

Reply:
I guess if we consider actual code changes we could just have a single list[str] attribute and then a way to say "does this DelegatedRole use hash prefixes Y/N?", but however you implement it (a flag attribute or even two separate classes) it it's going to look quite different compared to the file format ... so does not seem worth it

I think this might be worth pointing out in the body text of this class docstring... I'll try that